live-build/misc/migration-scripts/dx_execute

```diff
@@ -97,6 +97,21 @@ LX_RDS="$LX_RDS_PARENT/root"
 	die "the expected Linux RDS ($LX_RDS) was either not found or has" \
 		"been specified more than once in the bootloader's menu file"
 
+#
+# Replace the empty /var/delphix dataset from the migration image by a clone
+# of the current /var/delphix dataset.
+#
+LX_VAR_DLPX="$LX_RDS_PARENT/data"
+LX_CONTAINER="${LX_RDS_PARENT##*/}"
+CUR_VAR_DLPX=$(mount | awk '/^\/var\/delphix /{ print $3 }')
```

Comment:
would it be better to use `zfs list` here? e.g.
```
$ zfs list -Ho name /var/delphix
rpool/versions/2017.6.2017.06.28/dlpx/2017.6.28.18/running/var
```

Reply:
Cool, I didn't realize zfs list could take mountpoints as arguments.